window_test.go

```diff
@@ -65,6 +65,9 @@ func TestWindow_Actions(t *testing.T) {
 	var d = newDisplay(&DisplayOptions{Bounds: &RectangleOptions{PositionOptions: PositionOptions{X: astikit.IntPtr(1), Y: astikit.IntPtr(2)}, SizeOptions: SizeOptions{Height: astikit.IntPtr(1), Width: astikit.IntPtr(2)}}}, true)
 	testObjectAction(t, func() error { return w.MoveInDisplay(d, 3, 4) }, w.object, wrt, "{\"name\":\""+EventNameWindowCmdMove+"\",\"targetID\":\""+w.id+"\",\"windowOptions\":{\"x\":4,\"y\":6}}\n", EventNameWindowEventMove)
 	testObjectAction(t, func() error { return w.Resize(1, 2) }, w.object, wrt, "{\"name\":\""+EventNameWindowCmdResize+"\",\"targetID\":\""+w.id+"\",\"windowOptions\":{\"height\":2,\"width\":1}}\n", EventNameWindowEventResize)
+	bounds, err := w.Bounds()
```

Comment:
Could you update tests for functions that will update bounds as well?

- make sure to return a `bounds` key in the terminated event
- make sure to test `Bounds()` return the proper values

Reply:
> Could you update tests for functions that will update bounds as well?
> 
> * make sure to return a `bounds` key in the terminated event
> * make sure to test `Bounds()` return the proper values

I can add a test for `Move` function, but for others like `Maximize`, `ResizeContent`, I'm not sure of how to determine its returned bounds in the events.

Reply:
for other actions like `Maximize`, tests simulate what's returned by `astilectron` therefore all you need to do is add a `bounds` key with proper value

Reply:
I can't find where to attach the `bounds` in the returned event, do I have to modify `testObjectAction` to allow passing in a customized return event?

Reply:
I agree you have to modify `testObjectAction` but I would change its signature to this instead:

```go
func testObjectAction(t *testing.T, fn func() error, o *object, wrt *mockedWriter, sentEvent, eventNameDone string, receivedEvent *Event)
```

You'll have to add `, nil)` for other tests but that'd be more explicit.

Reply:
Done